test/puppetlabs/puppetdb/acceptance/cli.clj

```diff
@@ -0,0 +1,167 @@
+(ns puppetlabs.puppetdb.acceptance.cli
+  (:require [clojure.test :refer :all]
+            [puppetlabs.puppetdb.client :as pdb-client]
+            [puppetlabs.puppetdb.cli.export :as cli-export]
+            [puppetlabs.puppetdb.cli.import :as cli-import]
+            [puppetlabs.puppetdb.cli.anonymize :as cli-anon]
+            [puppetlabs.puppetdb.anonymizer :as anon]
+            [puppetlabs.puppetdb.testutils :as tu]
+            [puppetlabs.puppetdb.testutils.facts :as tuf]
+            [puppetlabs.puppetdb.testutils.reports :as tur]
+            [puppetlabs.puppetdb.testutils.catalogs :as tuc]
+            [puppetlabs.puppetdb.testutils.services :as svc-utils]
+            [puppetlabs.puppetdb.testutils.cli :refer :all]
```

Comment:
Alright, I'm a fan of this being aliased, sounded like @rbrw and @wkalt also agreed